src/components/badges/BadgeBlock.tsx

```diff
@@ -64,12 +67,12 @@ function Badge({ proof }: { proof: BaseProof }) {
           <>
             {proof instanceof ERC721Proof ? (
               <ContractName address={proof.contract} />
-            ) : proof instanceof EmailProof ? (
+            ) : isEmailProof ? (
               proof.domain
             ) : (
               'Unknown'
             )}{' '}
-            (derivative)
+            {isEmailProof ? '(email)' : '(derivative)'}
```

Comment:
I think keeping `derivative` is better?

Reply:
Inside the design, we have `(email)`
![image](https://user-images.githubusercontent.com/49658988/175763274-ebd76761-7acc-454a-92fc-d14ffee82cf9.png)